code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -37,7 +37,7 @@
 	var/bot_name
 
 	var/list/player_access = list() //Additonal access the bots gets when player controlled
-	var/emagged = FALSE
+	var/emagged = FALSE  //if 1, then a lockhacker was used. If 2, then an emag was used.

```

Comment:
Make these defines in [code/__DEFINES/misc.dm](https://github.com/tgstation/tgstation/blob/511e357472fe75be261f904bce00c215ea71ed76/code/__DEFINES/misc.dm), e.g.
```DM
#define EMAGGED_LOCKHACK 1
#define EMAGGED_EMAG 2
```
then usage would be as so: ```if(emagged == EMAGGED_EMAG)``` or ```if(emagged == EMAGGED_LOCKHACK)```

Reply:
Will be changing those tomorrow.

Reply:
Wish there was a way for me to NOT make this one outdated, since I still have to do this.